resolver/resolver.go

```diff
@@ -2547,6 +2548,8 @@ func (r *Resolver) resolveVariableExprCELValues(ctx *context, env *cel.Env, expr
 		for detIdx, detail := range expr.Validation.Error.Details {
 			r.resolveMessageValidationErrorDetailCELValues(ctx, env, ctx.msg, ctx.defIndex(), detIdx, detail)
 		}
+		// This is a dummy type since the output from the validation is not supposed to be used (at least for now)
+		expr.Type = BoolType
```

Comment:
This is a bit hacky, but I needed to set some type for MessageResolver#ZeroValue, which is called [here](https://github.com/mercari/grpc-federation/blob/ffeb8a3c7e94ab319b2fa9de363aa7db2f4a5a0f/generator/templates/server.go.tmpl#L617)

Reply:
This type should be error type, but it is ok now